charts/argo-cd/templates/argocd-application-controller/statefulset.yaml

```diff
@@ -344,6 +344,12 @@ spec:
                 name: argocd-cmd-params-cm
                 key: controller.cluster.cache.events.processing.interval
                 optional: true
+          - name: ARGOCD_APPLICATION_CONTROLLER_COMMIT_SERVER
+            valueFrom:
+              configMapKeyRef:
+                name: argocd-cmd-params-cm
+                key: commit.server
+                optional: true
```

Comment:
Though Argo CD itself has interface for this, [ref](https://github.com/search?q=repo%3Aargoproj%2Fargo-cd%20ARGOCD_APPLICATION_CONTROLLER_COMMIT_SERVER&type=code), we follow [upstream's manifests](https://github.com/argoproj/argo-cd/tree/master/manifests/base). Could you please refactor the upstream's manifest at first? Once upstream releases it, we will also follow it.

Reply:
Right, thanks for the heads up - I've raised a PR with the required changes upstream: https://github.com/argoproj/argo-cd/pull/23536